src/coremods/core_oper/core_oper.h

```diff
@@ -41,11 +33,18 @@ namespace DieRestart
  */
 class CommandDie : public Command
 {
+ protected:
+	bool CheckPass(User* user, const std::string& inputpass) const;
+
  public:
+	std::string hash, password;
```

Comment:
`hash` is shared between both the DIE and RESTART commands so you should store it in the `Module` class and pass a reference to it in the constructor.